kong/db/migrations/core/023_360_to_370.lua

```diff
@@ -0,0 +1,13 @@
+return {
+  postgres = {
+    up = [[
+      CREATE TABLE IF NOT EXISTS "clustering_rpc_requests" (
+        "id"         BIGSERIAL                  PRIMARY KEY,
+        "node_id"    UUID                       NOT NULL,
+        "reply_to"   UUID                       NOT NULL,
+        "ttl"        TIMESTAMP WITH TIME ZONE   NOT NULL,
+        "payload"    JSON                       NOT NULL
```

Comment:
```suggestion
        "payload"    JSONB                      NOT NULL
```

Reply:
Yes, there are some penalty on using JSONB, but we may need perhaps its flexibility to debug something, or just have a better validation/normalization on JSON. I think we use everywhere else JSONB. Not a biggie.

This more or less concludes it is not that significant:
https://erthalion.info/2017/12/21/advanced-json-benchmarks/#jsonb-vs-json-vs-table

A summary:
<img width="692" alt="image" src="https://github.com/Kong/kong/assets/983257/156d56ca-429c-43fc-accb-38285798759c">

-- https://dev.to/ftisiot/json-vs-jsonb-in-postgresql-5cj3

Both are fine.

Reply:
Technically, I think we can get away with just `TEXT`... It is probably faster and is validation really necessary for this use case? It is not a user provided field anyway.